extensions/parquet/table/plugin/channelprovider/tracked/src/main/java/io/deephaven/parquet/table/plugin/channelprovider/tracked/TrackedSeekableChannelsProvider.java

```diff
@@ -1,12 +1,14 @@
 /**
  * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
  */
-package io.deephaven.parquet.table.util;
+package io.deephaven.parquet.table.plugin.channelprovider.tracked;
```

Comment:
Commenting here b/c it's easier - but this class and constructor shouldn't need to be public after this.